### DISCUSSION TITLE
SIWave DCIR Simulation Result Export

### DISCUSSION THREAD

**Discussion by slinke61** [ansys/pyaedt#4157] [Q&A]
Hi everyone,

is there a way to customize the exported report beyond the "export_siwave_report" function from the SIWave class? (https://github.com/ansys/pyaedt/blob/main/pyaedt/siwave.py#L312)

Specifically, I would like to automatically generate a report from a PCB DCIR simulation using a script. This report should contain an image on which the voltages of certain nets are displayed across all layout layers. 

So far I have only managed to create a report with the "export_siwave_report" function, which contains an image of each layer with the voltage of the defined nets.

**Comment by Samuelopez-ansys** [comment]
This does not belong anymore to PyAEDT, please if you still have problems go to PyEDB:

https://github.com/ansys/pyedb
